src/app/pages/viewing/single-message/single-message.component.html

```diff
@@ -0,0 +1,35 @@
+<div class="row justify-content-center mt-5">
+  <div class="col-12 col-sm-12 col-md-8 col-lg-6 col-xl-5">
+    <div class="card shadow">
+      <div class="msg-header">
+        <div class="msg-header-group">
+          <table class="header-table">
+            <tr>
+              <td>
+            <span class="profile-pic-container-small mr-2">
+              <img class="profile-pic" src="{{ viewService.singleMessage.fromPic }}" alt="profile-pic">
+            </span>
+              </td>
+              <td>
+                <table>
+                  <tr class="msg-header-text">
+                    {{ viewService.singleMessage.fromName }}
+                  </tr>
+                  <tr class="msg-header-date">
+                    {{ viewService.timeSince(viewService.nowMillis, viewService.singleMessage.timestamp.toMillis(), viewService.singleMessage.timestamp.toDate().toDateString()) }}
+                  </tr>
+                </table>
+              </td>
+            </tr>
+          </table>
+        </div>
+      </div>
+      <div class="card-body message-card message" [ngStyle]="{'background': viewService.singleMessage.backColor, 'color': viewService.singleMessage.fontColor, 'font-family': viewService.fonts[viewService.singleMessage.fontFamily].fontFamily +','+ viewService.fonts[viewService.singleMessage.fontFamily].backupFont}">
```

Comment:
Break line up into multiple lines for readability (github formatting is weird so I couldnt suggest a change directly sry)